index.bs

```diff
@@ -155,114 +155,120 @@ mean the same thing as:
 </emu-alg>
 
+<h3 id="common-operations">Built-in Function Objects</h3>
+
+We follow the ECMA-262 <a href="http://www.ecma-international.org/ecma-262/6.0/index.html#sec-built-in-function-objects">convention for built-in function objects</a> in which the value of NewTarget in each function argument is <b>**undefined**</b> for \[[Call]] and the <i>newTarget</i> parameter for \[[Construct]].
+
 <h2 id="loader-objects">Loader Objects</h2>
 
-<h3 id="loader-constructor">The Reflect.Loader Constructor</h3>
+<h3 id="loader-abstract-operations">Abstract Operations for Loader Objects</h3>
+
+<h4 id="loader-CreateRegistry" aoid="CreateRegistry">CreateRegistry()</h4>
+
+The abstract operation CreateRegistry with no arguments performs the following steps:
+
+<emu-alg>
+1. Let _O_ be OrdinaryCreateFromConstructor(Registry, "%RegistryPrototype%", «[[RegistryMap]]» ).
+1. ReturnIfAbrupt(_O_).
+1. Return _O_.
+</emu-alg>
+
+<emu-note>
+  This abstract operation helps to prevent creation of new registry objects in user-land.
+</emu-note>
 
-The Loader constructor is the initial value of the Loader property of the the Reflect object. When called as a constructor it creates and initializes a new Loader object. Reflect.Loader is not intended to be called as a function and will throw an exception when called in that manner.
+<h3 id="loader-constructor">The Loader Constructor</h3>
 
-The Reflect.Loader constructor is designed to be subclassable. It may be used as the value in an extends clause of a class definition. Subclass constructors that intend to inherit the specified Loader behaviour must include a super call to the Reflect.Loader constructor to create and initialize the subclass instance with the internal stage necessary to support the Reflect.Loader.prototype built-in methods.
+The Loader constructor is the initial value of the Loader property of the the Reflect object. When called as a constructor it creates and initializes a new Loader object. Loader is not intended to be called as a function and will throw an exception when called in that manner.
 
-<h4 id="new-reflect-loader">Reflect.Loader()</h4>
+The Loader constructor is designed to be subclassable. It may be used as the value in an extends clause of a class definition. Subclass constructors that intend to inherit the specified Loader behaviour must include a super call to the Loader constructor to create and initialize the subclass instance with the internal stage necessary to support the Loader.prototype built-in methods.
 
-When Reflect.Loader is called with no arguments, the following steps are taken:
+<h4 id="new-loader">Loader()</h4>
+
+When Loader is called with no arguments, the following steps are taken:
 
 <emu-alg>
 1. If NewTarget is *undefined*, then throw a *TypeError* exception.
-1. Let _O_ be OrdinaryCreateFromConstructor(NewTarget, "Reflect.Loader.prototype").
+1. Let _O_ be OrdinaryCreateFromConstructor(NewTarget, "Loader.prototype").
```

Comment:
we dont' have `"%LoaderPrototype%"` anymore. We have changed everything from `Reflect.Loader` to just `Loader` objects. I can bring `"%LoaderPrototype%"`.